calculadora_do_cidadao/fields.py

```diff
@@ -13,7 +13,23 @@ def deserialize(cls, value: str) -> Decimal:
         return super().deserialize(value.replace(",", "."))
 
 
+class UsDecimalField(fields.DecimalField):
+    """ Field for reading decimal fields in American format. """
+
+    @classmethod
+    def deserialize(cls, value: str) -> Decimal:
+        """Deserialize decimals using a comma as a decimal separator."""
+        value = value or ""
+        return super().deserialize(value)
+
+
 class DateField(fields.DateField):
     """Field for mmm/yyyy dates."""
 
     INPUT_FORMAT = "%b/%Y"
+
+
+class IsoDateField(fields.DateField):
+    """Field for mmm/yyyy dates."""
+
+    INPUT_FORMAT = "%Y-%m-%d"
```

Comment:
Why do we need that? Isn't that just Rows's default [`DateField`](https://github.com/turicas/rows/blob/develop/rows/fields.py#L358)?

Reply:
Removing it!